### ISSUE TITLE
New Tab button moves on hover

### ISSUE THREAD

**Issue by abhibeckert** [sequelpro/sequelpro#2472]
In the latest nightly, hovering over the "new tab" button moves it a few pixels.

**Comment by jaimeeee** [comment]
And it blinks too.

**Comment by abhibeckert** [comment]
The movement on hover is fixed now. Blinking is caused by poorly written third party code that we're using for the tab bar. It's not easy to fix.

OS X 10.12 adds native support for tab bars, at some point we'll adopt this and that will solve the flickering.

**Comment by dmoagx** [comment]
@abhibeckert Have you looked at how the tabs in 10.12 work?
I'm very tempted to say "Let's throw out all of our tab code even if it means no longer having tabs on older OS X versions" but I'm wondering about colored tabs.

**Comment by abhibeckert** [comment]
@dmoagx Yeah, see 13 minutes 30 seconds into https://developer.apple.com/videos/play/wwdc2016/203/

Each tab is just an NSWindow, with background tabs not being drawn to the screen (NSWindow.visible will be "true" even if it's a background tab).

If anything in the responder chain responds to a "new window for tab" action, then tabbing is enabled for the window, and no other code is necessary aside from activating non-default behaviour and handling special cases.

We should just disable our current tab code on 10.12, and remove it at some point in future when we stop requiring older releases.
